packages/contracts/src/EmailWalletCore.sol

```diff
@@ -208,12 +220,16 @@ contract EmailWalletCore is ReentrancyGuard, OwnableUpgradeable, UUPSUpgradeable
         bytes memory psiPoint,
         bytes memory proof
     ) public returns (Wallet) {
+        // TODO: What stops `walletSalt` from being bytes32(0)?
```

Comment:
The proof would fail so. Most inputs used in proofs are not "null checked". These functions are only called by relayer anyway and are expected to pass the correct proof.